Add intial service info docs to README (#1)
cloud-gov/aws-elasticsearch-example: migrator/README.md

Migrating Elasticsearch services with snapshots
===============================================

> The following with example CF deployment is an overview of migrating to the new AWS Elasticsearch service using a snapshot from the legacy Elasticsearch service.

Reviewer: Slight grammar tweak:

```suggestion
The following example CF deployment is an overview of migrating to the new AWS Elasticsearch service using a snapshot from the legacy Elasticsearch service.
```

## Getting started

> The following example uses three services to create, store, and restore an Elasticsearch snapshot. The following services created for this example are named `old-es-service` (legacy elasticsearch56 service), `new-es-service` (new aws-elasticsearch service), and `snapshot-s3-service` (AWS S3 storage service).  Before running through this example, make sure to create the three services with the following cli commands:

Reviewer: This suggested clarification may be a bit easier to read and more direct with the instructions:

```suggestion
The following example uses three services to create, store, and restore an Elasticsearch snapshot. In this example the services are named `old-es-service` (legacy elasticsearch56 service), `new-es-service` (new aws-elasticsearch service), and `snapshot-s3-service` (AWS S3 storage service).

For the first step, create the three services with the following CLI commands:
```

Good call on specifying the instance details to make it clear what these are!

```bash
##
## Creating the following Elasticsearch example services
##

### The legacy elasticsearch service
$ cf create-service elasticsearch56 medium old-es-service

### The new elasticsearch service
$ cf create-service aws-elasticsearch BETA-es-dev new-es-service
```

Create a new S3 service to store the snapshot from the Elasticsearch service

```bash
##
## Creating the following S3 example service
##

### Make sure to use the "basic" plan so the snapshot is not publicly available
$ cf create-service s3 basic snapshot-s3-service
```

> To get started with this example, please clone or download this repository so you can follow along and run the steps from your computer.

Reviewer: Since this is a continuation from the steps taken above, it might make sense to continue the flow:

```suggestion
To continue with this example, please clone or download this repository so you can follow along and run the steps from your computer.
```

```bash

> ## Clone and navitgate into example repository

Reviewer: Spelling/grammar nit:

```suggestion
## Clone and navigate into the example repository
```

$ git clone https://github.com/cloud-gov/aws-elasticsearch-example && cd aws-elasticsearch-example
```

## Deploying

> The following deployment instructions assumes that you have cloned or downloaded this `aws-elasticsearch-example` and have navigated into the root of this directory (`$ cd migrator/`).  From the `migrator` directory push the CF deployment to you cloud.gov space.
> 
> ```bash
> $ cf push -f manifest.yml

Reviewer: I think it's safe to assume the developer has been following along the steps, so we ought to be able to make this a bit more concise:

```suggestion
From the root directory of the repository that you should now be in (`aws-elasticsearch-example`), navigate to the `migrator` directory and push the CF deployment to your cloud.gov space:

```bash
# Navigate to the migrator sub-directory
$ cd migrator

# Push the application
$ cf push -f manifest.yml
```

Hopefully this suggestion isn't mangled due to the nested code blocks...

```

> The following apps are deployed into your space:
> - `elastic-migrator` is a CF app task runner used to migrate snapshot data and indices into the elasticsearch services.

Reviewer: This might be more clearly stated in this fashion:

```suggestion
When the `cf push` command finishes, the following apps are deployed into your cloud.gov space:
- `elastic-migrator` - an app task runner that is used to migrate snapshot data and indices into the Elasticsearch services.
```

I'm assuming more apps may be deployed as a part of this?  If not, then this might make more sense:

```suggestion
When the `cf push` command finishes an `elastic-migrator` app task runner will be deployed, which is used to migrate snapshot data and indices into the Elasticsearch services.
```

Author: Originally, I was thinking about hooking up a kibana dashboard and using some more robust data from https://github.com/elastic/examples.  It's probably too much for the example repo at this stage.
